src/groups/mqb/mqba/mqba_initialconnectionhandler.h

```diff
@@ -39,6 +40,7 @@ namespace mqba {
 
 // FORWARD DECLARATION
 class SessionNegotiator;
+class Authenticator;
```

Comment:
```suggestion
```

This forward declaration is not correct (`mqbnet` namespace needed) and not used (`#include <mqbnet_authenticator.h>` present)